src/interface/IMintFeeManager.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+interface IMintFeeManager {
+
+    function getPlatformFeeAndRecipient(uint256 _price) external view returns (uint256, address);
```

Comment:
nit: this function name is a little confusing. is the return (uint256) = the fee? the final price? bps?
if possible, don't use `platform` because it has a different meaning in other contracts.

Reply:
updated to now use the function name`calculatePlatformFeeAndReceipient` to imply that it will calculate the final platform fee based on the percentage taken

Renamed to now use `platformFee` and `platformFeeRecipients` respectively for the return statements

Reply:
can't really think of a good alternative name for "platform". The best alternatives I can think of are the following:
- system
- infrastructure
- ecosystem